langchain-interpreter/src/langchain_interpreter/interfaces/webhook.py

```
@@ -0,0 +1,629 @@
+    if auto_subscribe and subscription.hub and subscription.topic:
+        callback_url = subscription.callback or f"http://localhost:8000{webhook_path}"
+        websub_subscriber = WebSubSubscriber(
+            hub=subscription.hub,
+            topic=subscription.topic,
+            callback=callback_url,
+            secret=secret,
+        )
+
+    # Create lifespan context manager for startup/shutdown
+    @asynccontextmanager
+    async def lifespan(app: FastAPI) -> AsyncGenerator[None, None]:
```

Ruff (0.14.14)

[warning] Unused function argument: `app`

(ARG001)